lib/tests/2wp.js

```diff
@@ -115,6 +115,39 @@ const execute = (description, getRskHost) => {
 
     });
 
+    it('should do a basic legacy pegin with the value above minimum', async () => {
+
+      // Arrange
+
+      const initial2wpBalances = await get2wpInitialBalances();
+      const senderRecipientInfo = await createSenderRecipientInfo(rskTxHelper, btcTxHelper);
+      const initialSenderAddressBalanceInSatoshis = await getBtcAddressBalanceInSatoshis(btcTxHelper, senderRecipientInfo.btcSenderAddressInfo.address);
+      // Value above minimum
+      const peginValueInSatoshis = minimumPeginValueInSatoshis + Number(btcToSatoshis(0.1));
```

Comment:
```suggestion
      const peginValueInSatoshis = minimumPeginValueInSatoshis + 1;
```
Maybe make just above the minimum? To be sure